etor/src/main/java/gov/hhs/cdc/trustedintermediary/external/localfile/FilePartnerMetadataStorage.java

```diff
@@ -101,7 +101,9 @@ public Set<PartnerMetadata> readMetadataForSender(String sender)
             throws PartnerMetadataException {
         try {
             return getPartnerMetadata().stream()
-                    .filter(metadata -> metadata.sender().equals(sender))
+                    .filter(
+                            metadata ->
+                                    metadata.sendingFacilityDetails().namespace().equals(sender))
```

Comment:
The comparison should probably be with universalId

Reply:
yes